### DISCUSSION TITLE
Feature request: support lxml.etree._Element as edit_config input

### DISCUSSION THREAD

**Discussion by dmfigol** [scrapli/scrapli_netconf#50] [Ideas]
It is possible that the user already has `lxml.etree._Element` as an input. To avoid unnecessary serialization/de-serealization (with `tostring` / `fromstring`), it would be nice if `edit_config` would accept that input.

**Comment by carlmontanari** [comment]
Yeah defo a good idea. Will need to re look into where/how the payload is getting used and how to make it consistent across all the methods. But yea, 100% agree and will get it sorted one of these days!

**Comment by carlmontanari** [comment]
this is now supported (in develop) on the bare `rpc` method -- this means if you want to do an edit config w/ a lxml object you've previously built you would need to have the `edit-config` tags on the lxml element, but scrapli netconf will still handle the outer most rpc tag and the message id and all that stuff. for now thats where Ill leave this, but we can revisit when I have some cycles to have a deeper look at things and/or it comes up more and/or the rpc method is lame!
